source/components/namespace/nsdump.c

```diff
@@ -635,10 +635,6 @@ AcpiNsDumpOneObject (
             break;
         }
         break;
-
```

Comment:
Getting rid off of this default case under display type in nsdump.c is not necessary and it also throws a warning which we treat as error in Unix environment.